ldap-client.go

```diff
@@ -41,7 +45,10 @@ func (lc *LDAPClient) Connect() error {
 				return err
 			}
 		} else {
-			l, err = ldap.DialTLS("tcp", address, &tls.Config{InsecureSkipVerify: false})
+			l, err = ldap.DialTLS("tcp", address, &tls.Config{
+				InsecureSkipVerify: lc.InsecureSkipVerify,
```

Comment:
For my understanding, why doing that instead of passing `UseSSL` to `true`? 
And what does passing `ServerName` change?